includes/css/scss/_module-header-brand-dealership.scss

```
@@ -33,23 +23,13 @@
 	width: 100%;
 	max-width: $container-width;
 
-	@supports (grid-area: auto) {
-		width: auto;
-		gap: $gap-size;
-	}
-
 	&::before {
 		content: "";
 		height: 2px;
 		width: 3%;
```

Please change the `width` to an absolute value. There's no reason the logo should move depending on the width of the `viewport`. **On a side note**, the mobile version of the logo/brand line/dealer name will have to be sorted for mobile, but that's outside the scope of this PR.
```suggestion
		width: 40px;
```